docs/installation/application-configuration.md

```diff
@@ -52,3 +52,16 @@ These variables are specific for [shopsys/google-cloud-bundle](https://github.co
 | --------------------------------------- | ------- | -------------------------------------------------------- |
 | `GOOGLE_CLOUD_PROJECT_ID`               | `''`    | defines Google Cloud Project ID                          |
 | `GOOGLE_CLOUD_STORAGE_BUCKET_NAME`      | `''`    | defines Bucket Name in Google CLoud Storage              |
+
+
+### S3 Bridge Bundle
+
+These variables are specific for [shopsys/s3-bridge](https://github.com/shopsys/s3-bridge)
```

Comment:
I'm not sure, but the configuration is optional - a project does not need use S3. Do we want to add a note that this is an optional setting if the project doesn't want to use S3?